gem/templates/rwanda/profiles/reset_password_success.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block content %}
+<div class="profiles">
+  <div class="heading">
+    <h1 class="heading__profiles">{% trans "PIN Changed" %}</h1>
+  </div>
+  <div class="heading">
+    <h3 class="heading__subheading">{% trans "Your PIN has been changed sucessfully." %}</h3>
```

Comment:
Is there a difference between PIN and password?

Reply:
@sewagodimo yes there is, the convention is to use pin throughout the client facing content.